src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -0,0 +1,1571 @@
+	using FluidProperties::GetDensityGlycol;
+	using FluidProperties::GetSpecificHeatGlycol;
+	using ReportSizingManager::ReportSizingOutput;
+
+	// Data
+	//MODULE PARAMETER DEFINITIONS
+
+	std::string const cCMO_CoolingPanel_Simple( "ZoneHVAC:CoolingPanel:RadiantConvective:Water" );
+	// Control types:
+	int const MATControl( 1 ); // Controls system using mean air temperature
+	int const MRTControl( 2 ); // Controls system using mean radiant temperature
+	int const OperativeControl( 3 ); // Controls system using operative temperature
+	int const ODBControl( 4 ); // Controls system using outside air dry-bulb temperature
+	int const OWBControl( 5 ); // Controls system using outside air wet-bulb temperature
+	int const ZoneTotalLoadControl( 6 ); //controls system using zone total remaining load
+	int const ZoneConvectiveLoadControl( 7 ); //controls system using zone convective remaining load
+	// Condensation control types:
+	int const CondCtrlNone( 0 ); // Condensation control--none, so system never shuts down
+	int const CondCtrlSimpleOff( 1 ); // Condensation control--simple off, system shuts off when condensation predicted
+	int const CondCtrlVariedOff( 2 ); // Condensation control--variable off, system modulates to keep running if possible
+
+	// DERIVED TYPE DEFINITIONS
+
+	//MODULE VARIABLE DECLARATIONS:
+	int NumCoolingPanels( 0 );
+	Array1D< Real64 > CoolingPanelSource; // Need to keep the last value in case we are still iterating
+	Array1D< Real64 > CoolingPanelSrcAvg; // Need to keep the last value in case we are still iterating
+	Array1D< Real64 > ZeroSourceSumHATsurf; // Equal to the SumHATsurf for all the walls in a zone with no source
+
+	// Record keeping variables used to calculate CoolingPanelSrcAvg locally
+	Array1D< Real64 > LastCoolingPanelSrc; // Need to keep the last value in case we are still iterating
+	Array1D< Real64 > LastSysTimeElapsed; // Need to keep the last value in case we are still iterating
+	Array1D< Real64 > LastTimeStepSys; // Need to keep the last value in case we are still iterating
+	Array1D_bool CheckEquipName;
+	Array1D_bool SetLoopIndexFlag; // get loop number flag
+
+	//SUBROUTINE SPECIFICATIONS FOR MODULE Simple Chilled Ceiling Panel
+
+	// Object Data
+	Array1D< CoolingPanelParams > CoolingPanel;
+
+	// Functions
+
+	void
+	SimCoolingPanel(
+		std::string const & EquipName,
+		int const ActualZoneNum,
+		int const ControlledZoneNum,
+		bool const FirstHVACIteration,
+		Real64 & PowerMet,
+		int & CompIndex
+	)
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Rick Strand
+		//       DATE WRITTEN   Aug 2014
+		//       MODIFIED       na
+		//       RE-ENGINEERED  na
+
+		// PURPOSE OF THIS SUBROUTINE:
+		// This subroutine simulates the simple cooling (chilled ceiling) panel.  It borrows heavily
+        // from the hot water radiant-convective baseboard model code.
+
+		// METHODOLOGY EMPLOYED:
+		// See the hot water radiant-convective baseboard code.
+
+		// REFERENCES:
+		// na
+
```

Comment:
@RKStrand Please delete unused comment blocks like this.  Current style is minimal comments, don't need these template comments anymore.